File: src/DashboardUI/src/data-contracts/UserProfileCreateRequest.ts
import { UserStoreRequestBase } from './UserStoreRequestBase';


export interface UserProfileCreateRequest extends UserStoreRequestBase {
  $type: 'UserProfileCreateRequest';
  firstName: string;
  lastName: string;
  state: string;
  country: string;
  phoneNumber: string;
  affiliatedOrganization: string | null;
}

File: src/API/WesternStatesWater.WestDaat.Common/DataContracts/UserProfileCreateRequest.cs
namespace WesternStatesWater.WestDaat.Common.DataContracts;

public class UserProfileCreateRequest : UserStoreRequestBase
{
    public Guid UserId { get; set; }

    public string FirstName { get; set; } = null!;

    public string LastName { get; set; } = null!;

    public string UserName { get; set; } = null!;

    public string State { get; set; } = null!;

    public string Country { get; set; } = null!;

    public string PhoneNumber { get; set; } = null!;

    public string AffiliatedOrganization { get; set; } = null!;
}